templates/ssl.conf.j2

```diff
@@ -0,0 +1,8 @@
+[Service]
+{% if mesos_ssl_enabled %}
```

Comment:
move this condition to 

> if mesos_ssl_enabled

since this will create empty file with [service] definition